### DISCUSSION TITLE
Drag, Drop, Dance: A Proposal for Animated Interaction

### DISCUSSION THREAD

**Discussion by ivbrajkovic** [KevinVandy/mantine-react-table#79] [Ideas]
Hey there! Just had this idea and I think you'll like it. What if you put a fun twist on our drag and drop by adding some slick animations? Like when you drag an item, it gives off a cool, fluid motion, making it feel like you're physically moving stuff around. I think it'd give our app a bit of a 'wow' factor, plus make it more enjoyable to use. Can't wait to hear your thoughts on this!

Also, found a sample online that shows what I'm thinking. Check it out at the following link when you get a chance. 
[https://ui.mantine.dev/component/dnd-table#dnd-table](url)

**Comment by KevinVandy** [comment]
It would just have to have a low bundle size increase. Otherwise all for it.
